src/main.py

```diff
@@ -2,6 +2,8 @@
 # run this script once were good to go
 # A lot of try catch baby
 
+from xmlrpc.client import boolean
```

Comment:
Dont think this is nesscary. I believe Bool is built in to python so you should be able to delete this line